frontend/src/components/Review/ReviewCard/ReviewCard.js

```diff
@@ -86,8 +111,9 @@ export default connect(mapStateToProps, mapDispatchToProps)(ReviewCard);
 ReviewCard.propTypes = {
     author: PropTypes.string,
     author_id: PropTypes.number,
-    source: PropTypes.string,
+    paperSource: PropTypes.string,
```

Comment:
paperSource is only for searching results of papers. I think it should be deleted from irrelevant cards. Also, did you check the header of PaperCards on SearchResult?

Reply:
Yes, like I commented above there, a legacy code exists, which was originally using 'source' props, it may be a minor issue. There are some props which is not actually used now (as I think), maybe that made me confused. Anyway I'll revert it for not confusing others too.

Reply:
Well, I can't understand. Why do we still need 'source' even if you don't use it for subscription ? I think it is not legacy-code, but just thrown-out-code.

Reply:
deleted as you wish.